docs/docs/monitoring.md

```diff
@@ -0,0 +1,60 @@
+### Monitoring
+***
+Maxwell exposes certain metrics through either its base logging mechanism, JMX, HTTP or by push to Datadog. This is configurable through commandline options
+or the `config.properties` file. These can provide insight into system health.
+At present certain metrics are Kafka-specific - that is, not yet supported other producers.
+
+### Metrics
+All metrics are prepended with `MaxwellMetrics.`
```

Comment:
It'll be great if we can config this prefix, `MaxwellMetrics` can be a default value.